src/status_im/ui/screens/accounts/recover/views.cljs

```diff
@@ -35,62 +59,66 @@
       :error               (cond error (i18n/label error)
                                  warning (i18n/label warning))}]))
 
-(defview password-input [password error on-submit-editing]
-  (views/letsubs [inp-ref (atom nil)]
-    {:component-will-update
-     (fn [_ [_ new-password]]
-       (when (and (string? new-password)
-                  (string/blank? new-password)
-                  @inp-ref)
-         (.clear @inp-ref)))}
-    [react/view {:style                       styles/password-input
-                 :important-for-accessibility :no-hide-descendants}
-     [text-input/text-input-with-label
-      {:label               (i18n/label :t/password)
-       :accessibility-label :enter-password
-       :placeholder         (i18n/label :t/enter-password)
-       :default-value       password
-       :auto-focus          false
-       :on-change-text      #(re-frame/dispatch [:accounts.recover.ui/password-input-changed (security/mask-data %)])
-       :on-blur             #(re-frame/dispatch [:accounts.recover.ui/password-input-blured])
-       :secure-text-entry   true
-       :error               (when error (i18n/label error))
-       :on-submit-editing   on-submit-editing
-       :ref                 #(reset! inp-ref %)}]]))
+(defview passphrase-display [passphrase]
+  [react/text  {:style (styles/recovery-phrase-input true)} passphrase])
 
-(defview recover []
+(defview recoverA [cofx]
```

Comment:
We should not pass `cofx` to views, rather use a sub.

Reply:
Done!